[ADD] add module product_life_period

laetitia-gangloff opened the pull request:
The goal of this module is to define life period on product and set product in state end at the end of the period.

Cedric-Pigeon commented on product_life_period/i18n/fr.po:
```diff
@@ -0,0 +1,116 @@
+#: view:product.life.period:product_life_period.product_life_period_view_search
+#: view:product.life.period:product_life_period.product_life_period_view_tree
+msgid "Product Life Period"
+msgstr "Période de vie de l'articles"
```
Remove "s" on article

Cedric-Pigeon commented on product_life_period/__openerp__.py:
```diff
@@ -0,0 +1,36 @@
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
```
Please use simple headers following OCA templates
https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L3

Cedric-Pigeon commented:
:+1:

Cedric-Pigeon commented on product_life_period/views/product_life_period_views.xml:
```diff
@@ -0,0 +1,162 @@
+                                type="action"
+                                name="%(action_open_life_period_products)d"
+                                icon="fa-cubes">
+                                <field name="products_count" string="Products" widget="statinfo" />
```
It should be great to pass the current period in context as default value on product action

Commit:
[ADD] add module product_life_period
- define life period on product
- set product in state end at the end of the period

laetitia-gangloff commented:
@Cedric-Pigeon I add a context on action to set default period life

Commit:
[ADD] add setup for product_life_period
